repository-meta-analyzer/src/main/java/org/hyades/repositories/IMetaAnalyzer.java

```diff
@@ -81,4 +82,12 @@ public interface IMetaAnalyzer {
 
     String getName();
 
+    /**
+     * The component meta data to analyze for integrity.
+     * @param component the component to analyze
+     * @return an IntegrityMeta object
+     * @throws MetaAnalyzerException in case of any issue during metadata generation
```

Comment:
minor: seems like some comments crept in unintentionally. throws and version 3.1.0 :)